earthpy/tests/test_clip.py

```diff
@@ -63,6 +63,14 @@
 
 """ Run clip shape tests """
 
+def test_not_gdf():
+    """Test that an error is raised when provided with a non geodataframe
```

Comment:
Minor grammar/spelling check for this docstring -- spelling for GeoDataFrame, indefinite article with plural noun ("a non geodataframe objects")

```suggestion
    """Non-GeoDataFrame inputs raise attribute errors."""
```